Add documentation for new Enphase Envoy platform

jesserizzo opened the pull request:
**Description:**


**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#15081

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

jesserizzo committed:
Create documentation for Enphase Envoy component

jesserizzo committed:
Add list of available conditions

jesserizzo committed:
Upload Enphase logo

jesserizzo committed:
Add second example configuration.yaml

jesserizzo committed:
Fix minor formatting errors

jesserizzo committed:
Fix configuration variables section

jesserizzo committed:
Fix minor typo

frenck committed:
:pencil2: Removed duplicate header

frenck commented:
Looks great @jesserizzo!
I did remove the configuration header/title thingy, since that is already handled by our configuration tags.

✅ Approved.

Can be merged as soon as the parent PR gets merged.

MartinHjelmare commented on source/_components/enphase_envoy.markdown:
```diff
@@ -0,0 +1,59 @@
+footer: true
+logo: enphase.svg
+ha_category: Energy
+ha_release: 0.73
```
0.76

MartinHjelmare commented on source/_components/enphase_envoy.markdown:
```diff
@@ -0,0 +1,59 @@
+  required: false
+  type: string
+monitored_conditions:
+  description: The list of conditions to display in the front end
```
This doesn't have anything to do with the frontend per se. This decides what entities will be created, ie what data to collect.

MartinHjelmare commented on source/_components/enphase_envoy.markdown:
```diff
@@ -0,0 +1,59 @@
+comments: false
+sharing: true
+footer: true
+logo: enphase.svg
```
The filename here doesn't match the included file.

MartinHjelmare commented on source/_components/enphase_envoy.markdown:
```diff
@@ -0,0 +1,59 @@
+  description: The local IP Address of your Envoy
+  required: true
+  type: string
+name:
```
This isn't included in the config schema of the platform.

MartinHjelmare reviewed (changes requested):
The name of the markdown file is incorrect. It should be prefixed with `sensor.`.

jesserizzo committed:
Changed ip to ip_address

jesserizzo committed:
Minor fixes

Change title to sensor.enphase_envoy. Update version number. Fix typo in enphase logo file name.

MartinHjelmare commented on source/_components/sensor.enphase_envoy.markdown:
```diff
@@ -0,0 +1,59 @@
+  description: The local IP Address of your Envoy
+  required: true
+  type: string
+name:
```
Remove this key.

jesserizzo committed:
Remove name key from configuration

MartinHjelmare reviewed (approved):
Great!
